docs/api/synthetics/monitors/add-monitor-api.asciidoc

```diff
@@ -0,0 +1,246 @@
+
+`response.include_body`:: (Optional, boolean, default: true): Controls the indexing of the HTTP response body contents to the `http.response.body.contents` field.
+  * `include_body` ("on_error" | "never" | "always", default: on_error) Set response.include_body to one of the options listed below.
+         ** `on_error`: Include the body if an error is encountered during the check. This is the default.
+         ** `never`: Never include the body.
+         ** `always`: Always include the body.
+  * `include_body_max_bytes` (Optional, number, default: 1024) Set `response.include_body_max_bytes` to control the maximum size of the stored body contents.
+
+`check`:: (Optional, object): The check request settings.
+  * `request` An optional request to send to the remote host. Under check.request, specify these options:
+      ** `method` ("HEAD" | "GET" | "POST" | "OPTIONS"): The HTTP method to use. 
+      ** `headers` (Optional, object): A dictionary of additional HTTP headers to send. By default Synthetics will set the User-Agent header to identify itself.
+      ** `body` (Optional, string): Optional request body content.
+        
+      Example: This POSTs an x-www-form-urlencoded string to the endpoint
+[source,sh]
+--------------------------------------------------
+{
+  "check": {
+    "request": {
+      "method": "POST",
+      "headers": {
+        "Content-Type": "application/x-www-form-urlencoded"
+      },
+      "body": "name=first&email=someemail%40someemailprovider.com"
+    }
+  }
+}
+--------------------------------------------------
+
+  * `response` The expected response. Under `check.response`, specify these options:
+      ** `status`  A list of expected status codes. 4xx and 5xx codes are considered down by default. Other codes are considered up.
+      ** `headers` (Optional, object): A dictionary of expected HTTP headers. If the header is not found, the check fails.
+      ** `body.positive`: A list of regular expressions to match the body output. Only a single expression needs to match.
+          Example:
+          This monitor examines the response body for the strings foo or Foo:
+      ** `body.negative`: A list of regular expressions to match the body output negatively. Return match failed if single expression matches. HTTP response bodies of up to 100MiB are supported.
+          This monitor examines match successfully if there is no bar or Bar at all, examines match failed if there is bar or Bar in the response body:
+      ** `json`: A list of expressions executed against the body when parsed as JSON. Body sizes must be less than or equal to 100 MiB. 
+          *** `description` (string): A description of the check.
+          *** `expression` (string): The following configuration shows how to check the response using [gval](https://github.com/PaesslerAG/gval) expressions when the body contains JSON:
```

Comment:
```suggestion
          *** `expression` (string): The following configuration shows how to check the response using https://github.com/PaesslerAG/gval[gval] expressions when the body contains JSON:
```



Sorry, oops, this one isn't working correctly on the rendered page. I guess the asciidoc syntax is not the same for links. Looking at [the docs](https://docs.asciidoctor.org/asciidoc/latest/asciidoc-vs-markdown/#comparison-by-example) it seems like it might actually be this.